<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="RavinPC" failures="1" tests="4" name="PetTest" time="9.512" errors="0" timestamp="2025-07-23T09:52:10 IST">
  <testcase classname="api.tests.UserTest" name="testPostUserByName" time="3.488"/>
  <testcase classname="api.tests.UserTest" name="getUser" time="1.229"/>
  <testcase classname="api.tests.UserTest" name="updateUserByName" time="2.314"/>
  <testcase classname="api.tests.UserTest" name="deleteUserByName" time="1.234">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.UserTest.deleteUserByName(UserTest.java:83)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteUserByName -->
</testsuite> <!-- PetTest -->
